src/app/pages/LiquidityMining/components/HistoryTable/TableBody/index.tsx

```diff
@@ -1,32 +1,47 @@
+import { SkeletonRow } from 'app/components/Skeleton/SkeletonRow';
 import React from 'react';
 import { LiquidityPoolDictionary } from 'utils/dictionaries/liquidity-pool-dictionary';
 import { TableRow } from '../TableRow/index';
+import { useTranslation } from 'react-i18next';
+import { translations } from '../../../../../../locales/i18n';
+import { weiToFixed } from 'utils/blockchain/math-helpers';
 
-export const TableBody: React.FC = () => {
+interface ITableBodyProps {
+  items: any[];
+  loading: Boolean;
+}
+
+export const TableBody: React.FC<ITableBodyProps> = ({ items, loading }) => {
+  const { t } = useTranslation();
   const pools = LiquidityPoolDictionary.list();
 
   return (
     <tbody className="mt-5">
-      {pools.map((item, index) => (
-        <TableRow key={`${item.poolAsset}/${index}`} pool={item} />
+      {items.map((item, index) => (
+        <TableRow
+          key={`${item.poolAsset}/${index}`}
+          pool={pools[0]}
+          time={item.time}
+          txHash={item.txHash}
+          amount={weiToFixed(item.amount, 4)}
+          type={item.type}
+        />
       ))}
 
-      {/* {loading && (
-              <tr key={'loading'}>
-                <td colSpan={99}>
-                  <SkeletonRow
-                    loadingText={t(translations.topUpHistory.loading)}
-                  />
-                </td>
-              </tr>
-            )}
-            {history.length === 0 && !loading && (
-              <tr key={'empty'}>
-                <td className="text-center" colSpan={99}>
-                  History is empty.
-                </td>
-              </tr>
-            )} */}
+      {loading && (
+        <tr key={'loading'}>
+          <td colSpan={99}>
+            <SkeletonRow loadingText={t(translations.topUpHistory.loading)} />
+          </td>
+        </tr>
+      )}
+      {items.length === 0 && !loading && (
+        <tr key={'empty'}>
+          <td className="text-center" colSpan={99}>
+            History is empty.
```

Comment:
For this line, could you add the translation string in the same we way have for topupHistory - e.g.

`{t(translations.liquidityMining.historyTable.emptyState)}`

You will need to add this to `src/locales/en/translation.json` first, example is in object under `topUpHistory.emptyState` key